eigen/examples/svd/main/main.cpp

```diff
@@ -0,0 +1,57 @@
+/* Import Prebuilt Library Example
+
+   This example code is in the Public Domain (or CC0 licensed, at your option.)
+
+   Unless required by applicable law or agreed to in writing, this
+   software is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+   CONDITIONS OF ANY KIND, either express or implied.
+*/
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include <esp_log.h>
+
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
```

Comment:
Looks like this is unused? Please keep only the includes which the program actually needs.